client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -345,6 +347,39 @@ export default React.createClass( {
 				section="domain-management" />;
 	},
 
+	pendingTransfer() {
+		const domains = this.getDomains().filter( domain => domain.pendingTransfer );
+
+		if ( domains.length !== 1 ) {
```

Comment:
Don't think a comment is needed. This rule is not in the default whitelist, so it has to be passed through props.
 Add a line note